docs/README.md

```diff
@@ -23,6 +23,19 @@ $ cd ../website_playground
 $ CARGO_CFG_HYDROFLOW_GENERATE_DOCS="1" RUSTFLAGS="--cfg procmacro2_semver_exempt --cfg super_unstable" wasm-pack build
 ```
 
+### Notes on building on macOS
+If you're building on macOS, you may need to install the `llvm` package with Homebrew (because the default toolchain has WASM support missing):
+
+```bash
+$ brew install llvm
+```
+
+Then, you'll need to set `TARGET_CC` and `TARGET_AR` environment variables when building the playground:
+
+```bash
+$ TARGET_CC=/usr/local/opt/llvm/bin/clang TARGET_AR=/usr/local/opt/llvm/bin/llvm-ar CARGO_CFG_HYDROFLOW_GENERATE_DOCS="1" RUSTFLAGS="--cfg procmacro2_semver_exempt --cfg super_unstable" wasm-pack build
```

Comment:
should be 
```
TARGET_CC=$(brew --prefix)/opt/llvm/bin/clang TARGET_AR=$(brew --prefix)/opt/llvm/bin/llvm-ar CARGO_CFG_HYDROFLOW_GENERATE_DOCS="1" RUSTFLAGS="--cfg procmacro2_semver_exempt --cfg super_unstable" wasm-pack build
```

Reply:
quote `"$(brew --prefix)/opt/llvm/bin/clang"`

Reply:
quotes around `"$(brew --prefix)/opt/llvm/bin/clang"` `"$(brew --prefix)/opt/llvm/bin/llvm-ar"`